bc_obps/registration/tests/utils/baker_recipes.py

```diff
@@ -148,10 +149,9 @@
     OperationDesignatedOperatorTimeline,
     operation=foreign_key(operation),
     operator=foreign_key(operator),
-    status=cycle([status for status in OperationDesignatedOperatorTimeline.Statuses]),
     start_date=datetime.now(ZoneInfo("UTC")),
+    status=OperationDesignatedOperatorTimeline.Statuses.CLOSED,
```

Comment:
This is so that we can create more than one at a time with `_quantity=##`. Otherwise, we get an error about only one Active record allowed per operator/operation combo